root/components/events-list.tt

```diff
@@ -1,100 +0,0 @@
-<table class="tbl">
-  <thead>
-    <tr>
-      [%~ IF c.user_exists AND checkboxes ~%]
-        <th style="width: 1em">
-          <input type="checkbox" />
-        </th>
-      [%~ END ~%]
-
-      [%~ IF series_item_numbers ~%]
-        <th style="width: 1em">[%~ l('#') ~%]</th>
-      [%~ END ~%]
-      <th>[%~ sortable ? sortable_table_header('name', l('Event')) : l('Event') ~%]</th>
```

Comment:
This component accepted a "sortable" boolean. This was (unless I've missed some use I haven't touched at all, which I don't think so) completely unused, so I didn't implement it on the React component. We might eventually want these tables to be sortable, but then I don't see any reason why this would be done with a preference anyway - we'd want *all* tables to be sortable, I'd expect.

Reply:
We set `sortable=1` in root/collection/index.tt, so it does get used on collection pages from what I can see. I definitely agree we'd want all tables to be sortable, though at the moment it's not supported everywhere (since these lists are paginated, it has to be done on the server rather than the client - that's what the `?order=` query parameters on collection pages allow for).

Reply:
Oh, damn, I entirely missed the collection page (since it didn't come up with a search for the component filename). I guess we don't test collections work too well, given nothing complained :p 

So wait, how do I port the sorting, if it's currently done on the Perl level?

Reply:
Well firstly you gotta port the `sortable_table_header` macro and add back the `sortable` prop along with it. Then you don't need to touch anything on the server side, just set `sortable={true}` on collection pages.

Reply:
If you mean "how do I make the other pages sortable too," it'd need some changes to the controller/data layers which are prob outside the scope of this PR. :) For collections, we take an `order` query parameter and pass it to `find_by_collection` [here](https://github.com/reosarevok/musicbrainz-server/blob/b0a09f5/lib/MusicBrainz/Server/Controller/Collection.pm#L91:L96), which constructs an appropriate `ORDER BY` entry into the query. So we'd need to do something similar in the relevant Data:: methods these other pages use.